package-support-policy.md

```diff
@@ -0,0 +1,90 @@
+# Support Policy for .NET Packages
+
+NuGet packages owned by the .NET team are supported as long as they target at least one framework that is still actively supported.
+
+The minimum supported .NET versions are:
+
+- .NET Framework 4.6.2
+- .NET Standard 2.0
+- .NET 6.0
+
+The support policy of our frameworks can be found here: https://dotnet.microsoft.com/en-us/platform/support/policy
+
+
+## Package and project compatibility
```

Comment:
This text is quite verbose. It occurs to me that the more terse text in our exist support doc almost says the same thing: https://github.com/dotnet/core/blob/main/release-policies.md#end-of-support

Reply:
I see. I can rewrite the section to just point to that document so users can consult the way packages are supported depending on the project and the package tfms.

Reply:
I propose that we update that text to be a tad more expansive and then see if we need this document at all.